src/jailer/src/env.rs

```diff
@@ -454,6 +500,16 @@ mod tests {
         args.parse(&make_args(&invalid_node_arg_vals)).unwrap();
         assert!(Env::new(&args, 0, 0).is_err());
 
+        let invalid_cgroup_arg_vals = ArgVals {
```

Comment:
Well, currently there are no checks of duplicate cgroups, if a cgroup is duplicate it just rewrites the value each time and the last one will be the persistent. I think this way we do not add overhead for the "best case" and improving the "worst case" will be up to the user.

Reply:
> Well, currently there are no checks of duplicate cgroups, if a cgroup is duplicate it just rewrites the value each time and the last one will be the persistent. I think this way we do not add overhead for the "best case" and improving the "worst case" will be up to the user.

I am OK with this. 👍 

> 
> Regarding testing, maybe we can add a dedicated unit tests as you suggest for checking the format parsing and to make sure it will fails if something with incorrect format is passed (not just `zzz`).

It would be great if you can provide a test like this.

Reply:
Sure, I'll add the test in a commit with all the other requested changes.